frog/imports/api/activities.js

```diff
@@ -14,13 +15,45 @@ export const addActivity = (activity_type, data, id) => {
       _id: uuid(), 
       activity_type: activity_type, 
       data: data, 
-      created_at: new Date() })
+      created_at: new Date(),
+      status: 'OUT'
+    })
   }
 } 
 
 export const duplicateActivity = (activity) =>
   Activities.insert({...activity, _id: uuid(), data: {...activity.data, name: activity.data.name + ' (copy)'}})
 
+export const addGraphActivity = (params) => {
+  const id = uuid()
+  Activities.insert({...params, created_at: new Date(), _id:id, status: 'IN'})
+  return(id)
+}
+
+export const addGraphOperator = (params) => {
+  const id = uuid()
+  Operators.insert({...params, created_at: new Date(), _id:id, status: 'IN'})
+  return(id)
+}
+
+export const copyActivityIntoGraphActivity = (graphActivityId, fromActivityId) => {
+  const fromActivity = Activities.findOne({_id:fromActivityId})
+  Activities.update(graphActivityId, {$set: {data: fromActivity.data, activity_type: fromActivity.activity_type}})
```

Comment:
Maybe we should have a parent tag that points to where the activity was copied from? So we can say "someone updated this activity, merge to graph etc" in the future?